common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1006,10 +1053,10 @@ def check_problem(self, data, override_time=False):
         # Problem submitted. Student should reset before checking again
         if self.done and self.rerandomize == RANDOMIZATION.ALWAYS:
             event_info['failure'] = 'unreset'
-            self.track_function_unmask('problem_check_fail', event_info)
+            self.track_function_unmask('problem_submit_fail', event_info)
```

Comment:
This is the "title" for an event that will be published. Let's not change it to avoid any analytics fallout (and note that it appears in other places, such as line 1048).